.github/docker/centreon-web/bookworm/entrypoint/container.sh

```
@@ -7,5 +7,17 @@ set -x
 # ls is required to ensure that the scripts are properly sorted by name.
 BASEDIR="/usr/share/centreon/container.d"
 for file in `ls $BASEDIR` ; do
-  . "$BASEDIR/$file"
+  case "$file" in
+    *_background*)
+      # Execute background script and store PID
+      . "$BASEDIR/$file" &
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)